mlos_bench/mlos_bench/config/optimizers/manual_opt.jsonc

```
@@ -0,0 +1,11 @@
+// Mock optimizer to test the benchmarking framework.
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/MLOS/main/mlos_bench/mlos_bench/config/schemas/optimizers/optimizer-schema.json",
+
+    "class": "mlos_bench.optimizers.ManualOptimizer",
+
+    "config": {
+        "max_suggestions": 30,
```

```suggestion
        "max_cycles": 30,
```

(and the internally limit by `max_suggestions` since I think that's part of the base class config)

---

My thought is it makes intuitive sense to me to talk about testing the entire set of configs N times, rather than stop part way through.  Else it would ruin your comparisons later on.

---

`max_cycles` is a good idea, will implement it!